### ISSUE TITLE
Better auto complete for constants

### ISSUE THREAD

**Issue by v0dro** [minad/iruby#48]
In the command line interface of iruby, if I try to complete a constant after the `::`, it shows the previous constant too, it the autocompleted line.

Heres an example:

```
In [2]: Nyaplot::
Display all 121 possibilities? (y or n)

In [2]: Nyaplot::Nyaplot::
```

Even strange is this:

```

In [2]: Nyaplot::Da <tab>
Nyaplot::DataBase   Nyaplot::DataFrame  

In [2]: Nyaplot::Nyaplot::DataFrame 

```

**Comment by domitry** [comment]
Yes, that is one of the known problems of IRuby, and you can solve it by installing `Pry` as 'gem install pry'.

I had tried to fix it before and found that we have to add some lines to `Bond` gem or add complex meta-programming code to IRuby itself. I think specifying Pry as a dependent library in gemspec is better, but what do you think @minad?

**Comment by v0dro** [comment]
I have pry installed. It persists.

Will try fixing it tomorrow.

**Comment by domitry** [comment]
hmm I cannot reproduce your problem x-(
how is the code as follows?
```
In [0]: hoge = Nyaplot::Da [tab]
```

**Comment by v0dro** [comment]
Strangely, I'm not getting any autocomplete options when I use your code!

**Comment by minad** [comment]
Is this only a console or a notebook problem?

**Comment by minad** [comment]
Closed in favor of Sciruby/iruby#5
